src/DotNet.Status.Web/Controllers/TelemetryController.cs

```diff
@@ -0,0 +1,78 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Microsoft.DotNet.Kusto;
+using Kusto.Ingest;
+using System.Collections.Generic;
+
+namespace DotNet.Status.Web.Controllers
+{
+    [Route("api/telemetry")]
+    [ApiController]
+    public class TelemetryController : ControllerBase
+    {
+        private readonly ILogger<TelemetryController> _logger;
+        private readonly IOptionsSnapshot<TelemetryOptions> _options;
+        private readonly IKustoIngestClient _client;
+
+        public TelemetryController(
+            ILogger<TelemetryController> logger,
+            IOptionsSnapshot<TelemetryOptions> options,
+            IKustoIngestClient client = null)
+        {
+            _logger = logger;
+            _options = options;
+            _client = client;
+        }
+
+        [HttpPost("collect/ArcadeValidation")]
+        public async Task<IActionResult> CollectArcadeValidation([Required] ArcadeValidationData data)
+        {
+            _logger.LogInformation("Start Collect Arcade Validation data");
```

Comment:
This start and end log don't really add any value from what I can see. The request is already logged, and the action method invocation is also logged by MVC, logs in this method should log interesting data about the request, rather than just saying "i'm doing the thing that MVC just told you i'm doing"